### PR TITLE
update translation RULES, fix typo and fix title link

### PR DESCRIPTION
**Pull request by yoga1234**
**What is the purpose of this pull request? (put an "X" next to item)**

[ x] Documentation update
[x ] Bug fix
[ x] New feature
[ ] Other, please explain:

**What changes did you make? (Give an overview)**
Updating translation for Bahasa Indonesia (Indonesian language)

**Which issue (if any) does this pull request address?**
There some error on title link, and typo

**Is there anything you'd like reviewers to focus on?**
The translation for Bahasa Indonesia maybe.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,7 +22,7 @@
   <a href="/docs/README-en.md">English</a> •
   <a href="/docs/README-esla.md">Español (Latinoamérica)</a> •
   <a href="/docs/README-fr.md">Français</a> •
-  <a href="/docs/README-id.md">Indonesian</a> •
+    <a href="/docs/README-id.md">Bahasa Indonesia</a> •
```
Comment (LinusU) [inline README.md:25]:
```suggestion
  <a href="/docs/README-id.md">Bahasa Indonesia</a> •
```

File: .vscode/launch.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  "configurations": [
+    {
+      "type": "pwa-chrome",
+      "request": "launch",
+      "name": "Launch Chrome against localhost",
+      "url": "http://localhost:8080",
+      "webRoot": "${workspaceFolder}"
+    }
+  ]
+}
```
Comment (LinusU) [inline .vscode/launch.json:15]: I'm not sure if we want this checked in 🤔 

What is it, and what does it do?

File: README.md (hunk 1 above)
Comment (yoga1234) [reply README.md:25]: Oops..

File: .vscode/launch.json (hunk 2 above)
Comment (yoga1234) [reply .vscode/launch.json:15]: Oops again..

Sorry, I don't see this. Gonna come back :D

**Pull request closed without merging**